src/style.css

```diff
@@ -1,3 +1,82 @@
 body {
+  background-color: green;
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-bg-color: #00ACDC;
+}
+
+html {
+font-family: Roboto, sans-serif;
+}
+
+.header {
+  box-sizing: border-box;
+  background-color: white;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)